src/Compilers/CSharp/Test/Symbol/Symbols/Source/UsingAliasTests.cs

```diff
@@ -73,12 +71,124 @@ partial class A : Object {}
             Assert.Null(alias4);
         }
 
+        [Fact]
+        public void GetSemanticInfo_PrimitiveType()
+        {
+            var text =
```

Comment:
Multiple of these tests seem to share the same source. Consider consolidating. 
Alternatively, consider clarifying what is the purpose of each test. I couldn't tell at a glance.

Reply:
so i was basically taking all existing tests for using-aliases and producing a version that then aliased a non-named-type.  I was presuming that the existing tests were testing interesting stuff.  So now the new tests would do the same, just with coverage for different types :)